eventing/samples/gcp-pubsub-source/subscriber.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         spec:
           container:
             # This corresponds to
-            # https://github.com/knative/eventing-sources/blob/v0.2.1/cmd/message_dumper/dumper.go.
+            # https://github.com/knative/eventing-sources/blob/release-0.3/cmd/message_dumper/dumper.go
             image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper@sha256:ab5391755f11a5821e7263686564b3c3cd5348522f5b31509963afb269ddcd63
```

Comment:
Does the gcr.io link need updating?

Reply:
from the sha it's hard to tell, but I assume the image still works since there's not been changes to it. @Harwayne 

We should also consider tagging the images in our release processes with something that's easier to grok than sha.

Reply:
Thanks!